src/WebJobs.Script/Description/DotNet/DotNetScriptTypes.cs

```diff
@@ -5,6 +5,7 @@ namespace Microsoft.Azure.WebJobs.Script.Description
 {
     public class DotNetScriptTypes
     {
+        public const string Codeless = "Codeless";
```

Comment:
I'm thinking we should be able to do away with this new "language" type. For the LA scenario, the language type should just be DotNetAssembly.  If we need to continue to discriminate internally functions that we loaded from custom providers, rather than use language type, we could just have an internal property on FunctionMetadata that we set for functions we get from custom providers.

Reply:
Discussed offline, and updated with changes that addresses some of these concerns.